sympy/core/function.py

```diff
@@ -2349,7 +2349,7 @@ def count_ops(expr, visual=False):
                     ops.append(o*(len(a.args) - 1))
                 else:
                     ops.append(o)
-            if not a.is_Symbol:
+            if not a.is_Symbol and not a.is_MatrixSymbol:
```

Comment:
I think we should catch MatrixSymbol here. For example, should count_ops(M[0, i + 1]) = 1 or 0. If the answer is 1, we must catch MatrixSymbol and not MatrixElement